src/functions/Output.ps1

```diff
@@ -507,7 +511,7 @@ function Get-WriteScreenPlugin ($Verbosity) {
 
             if ("file" -eq $Context.Block.BlockContainer.Type) {
                 # write two spaces to separate each file
-                & $SafeCommands["Write-Host"] -ForegroundColor Magenta "`nRunning tests from '$($Context.Block.BlockContainer.Item)'"
+                & $SafeCommands["Write-Host"] -ForegroundColor $ReportTheme.TestRunBlock "`nRunning tests from '$($Context.Block.BlockContainer.Item)'"
```

Comment:
This is a container run. Can we name it ContainerRun, or just Container?

Reply:
renamed to container